ServerCore/Pages/Teams/TeamCompositionSummary.cshtml.cs

```diff
@@ -71,7 +72,8 @@ public TeamComposition(int teamId)
         public int TotalNonMicrosoftCount { get; set; }
         public int TotalTotal { get; set; }
         public int TotalPossibleEmployeeAliasesCount { get; set; }
-        
+        public bool TeamIsRemote { get; set; }

```

Comment:
It doesn't look like this is getting used. It would be helpful to have a count like the other counts here though